openlcs/openlcs/settings.py

```diff
@@ -320,4 +310,13 @@
             "CONNECTION_POOL_KWARGS": {"max_connections": 100}
         },
     },
-}
\ No newline at end of file
+}
+
+try:
+    # pylint:disable=wildcard-import,unused-wildcard-import
+    parent_dir = os.path.abspath(os.path.dirname(__file__))
+    if parent_dir not in sys.path:
+        sys.path.append(parent_dir)
+    from settings_local import *  # noqa
+except ImportError:
+    pass
```

Comment:
The change cannot work, because `REDIS_CACHE_LOCATION` is defined in the `settings_local.py`, L307 should read it first

Reply:
isn't `REDIS_CACHE_LOCATION` already defined in L302? Also If you want customization in settings local, you should include L302-L313.

I'm not sure I understand what you mean.  Please clarify.

Reply:
Actually in MP+ envs, we use the `settings_local.py` from the ansible playbook templates: https://gitlab.cee.redhat.com/pelc/openlcs-playbooks/-/commit/c530c3c64454522365a2ebdc3f15771f1c90b62a

but in this code, it will try use the following setting at first,
```
REDIS_CACHE_LOCATION = os.environ.get('REDIS_CACHE_LOCATION',
                                      'redis://localhost:6379/1')
```
therefore `REDIS_CACHE_LOCATION` of `CACHES` becomes `redis://localhost:6379/1`, and when attempting yo connect Redis, it turns out `connection refused` error. 

@rhyw @hanchuntao

Reply:
Because in MP+ envs, we should connect to `REDIS_CACHE_LOCATION = 'redis://redis:6379/1'`

Reply:
You should not add any addtional conf after the `from settings_local import *`, as all conf after that could overwrite what's defined in `settings_local`, which is **counter design**.

You should instead in the ansible template to include everying you want to customize. In this case, also add the `CACHES`.